### ISSUE TITLE
Can't download images from Facebook after call FirebaseApp.configure() - when app linked to FirebasePerformance.

### ISSUE THREAD

**Issue by vgorloff** [firebase/firebase-ios-sdk#486]
### [READ] Step 1: Are you in the right place?

When i'm calling method for image download before `FirebaseApp.configure()` it works.
When i'm calling method for image download after  `FirebaseApp.configure()` - getting iOS Networking error `finished with error - code: -1001`.

Line above `FirebaseApp.configure()`  - OK.
Line below `FirebaseApp.configure()` - Not OK.

### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 9.1 (AppStore) or 9.2 beta (9C34b)
  * Firebase SDK version: 4.6.0

#### Steps to reproduce:

- Use sample URL: `https://graph.facebook.com/v2.11/109693813139899/picture?type=square&width=140&height=140`
- Download image from sample URL before calling `FirebaseApp.configure()`.
As result image downloaded

- Download image from sample URL after calling `FirebaseApp.configure()`
As result iOS Networking error: `Task <ADC5BE66-ACD8-49FD-9863-D37AAEF25C31>.<1> finished with error - code: -1001`

#### Relevant Code:

```
class SampleDownloader {
   
   let urlSession = URLSession(configuration: URLSessionConfiguration.default)
   var task: URLSessionTask?
   let url = URL(string: "https://graph.facebook.com/v2.11/109693813139899/picture?type=square&width=140&height=140")!

   func download() {
      print(url)
      task = urlSession.dataTask(with: url) { data, response, error in
         print(data)
         print(response)
         print(error)
      }
      task?.resume()
   }
}

// ---
let sd = SampleDownloader()
sd.download() // Works
FirebaseApp.configure()

// ---
let sd = SampleDownloader()
FirebaseApp.configure()
sd.download() // `Task <...>.<1> finished with error - code: -1001`

```

**Comment by morganchen12** [comment]
Apple's docs say this is a [timeout error](https://developer.apple.com/documentation/cfnetwork/cfnetworkerrors/1416962-cfurlerrortimedout).

Does replacing your download link with a small file hosted locally make the issue go away? If so, it's likely not a Firebase issue.

**Comment by vgorloff** [comment]
Interesting that same URLSession used to download other images. Issue observed only when downloading image from Facebook. And only if Firebase app is configured (by calling `FirebaseApp.configure()`). :0

**Comment by vgorloff** [comment]
Curl working well: `curl -v -L --url 'https://graph.facebook.com/v2.11/109693813139899/picture?type=square&width=140&height=140'`

**Comment by vgorloff** [comment]
Same thing on Firebase 4.3.0. :0

**Comment by vgorloff** [comment]
Firebase 4.3.0:
Not linking to `FirebasePerformance` solved issue .)

Flags removed: `-framework FirebasePerformance -framework GTMSessionFetcher`

So issue in `FirebasePerformance` module.

**Comment by vgorloff** [comment]
Theoretically issue related to FirebasePerformance + `HTTP2`. Because Facebook CDN uses `HTTP2`.

**Comment by morganchen12** [comment]
I see, thanks for digging into that.

**Comment by vgorloff** [comment]
Firebase 4.6.0, Xcode 9.2 beta (9C34b), iOS 11.2:

Not linking to `FirebasePerformance` solved issue .)
Flags removed: `-framework FirebasePerformance -framework GTMSessionFetcher`

So issue definitely in `FirebasePerformance` module.

**Comment by morganchen12** [comment]
Filed internal but b/69680733; I'll keep this ticket updated as progress is made.

**Comment by vgorloff** [comment]
Thanks! .)

**Comment by mikehaney24** [comment]
This will be fixed in the next version of the SDK. Leaving open until then.

**Comment by votson** [comment]
So I should be good now

**Comment by votson** [comment]
So I be getting better now

**Comment by mikehaney24** [comment]
This issue should be fixed in the latest FirebasePerformance pod, v1.1.0. Please feel free to reopen if that's not the case!
